Engine/plugin/agsplugin.cpp

```diff
@@ -1016,10 +1016,9 @@ Engine::GameInitError pl_register_plugins(const std::vector<Common::PluginInfo>
             apl->filename = "ags_snowrain";
         }
 
-        String expect_filename = apl->library.GetFilenameForLib(apl->filename);
         if (apl->library.Load(apl->filename))
         {
-          AGS::Common::Debug::Printf(kDbgMsg_Info, "Plugin '%s' loaded as '%s', resolving imports...", apl->filename.GetCStr(), expect_filename.GetCStr());
+          AGS::Common::Debug::Printf(kDbgMsg_Info, "Plugin '%s' loaded as '%s', resolving imports...", apl->filename.GetCStr(), apl->library.GetFilePath().GetCStr());
```

Comment:
In the string, the meaning changed from "loaded as {expect_filename}" to "loaded from {file path}", so better use the word from there.